libpod/networking_linux.go

```diff
@@ -981,3 +983,139 @@ func (w *logrusDebugWriter) Write(p []byte) (int, error) {
 	logrus.Debugf("%s%s", w.prefix, string(p))
 	return len(p), nil
 }
+
+// DisconnectContainerFromNetwork removes a container from its CNI network
+func (r *Runtime) DisconnectContainerFromNetwork(nameOrID, netName string, force bool) error {
+	ctr, err := r.LookupContainer(nameOrID)
+	if err != nil {
+		return err
+	}
+
+	networks, err := ctr.networksByNameIndex()
```

Comment:
Could I convince you to take everything from here on, and turn it into `func (c *Container) NetworkDisconnect(...)` and then have this function call that?

Reply:
@mheon has agreed to allow me to defer this until my next PR